scripts/utils/create-jest-config.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+module.exports = (resolve, rootDir) => {
+  const config = {
+    automock: false,
```

Comment:
i really like that too.
@cpojer i though the default `automock: false` was in the plan for `14.0`, no?